src/components/Expense/AddBorrower.jsx

```diff
@@ -0,0 +1,40 @@
+import { useState, useContext } from "react";
+
+import { userContext } from "../../App";
+
+const AddBorrower = (props) => {
```

Comment:
should extract `setDataBorrower` method here
```suggestion
const AddBorrower = ({ setDataBorrower }) => {
```